### PR TITLE
If is a prop, can't be a data.

### PR DESCRIPTION
**Pull request by vitormattos**
Removed src from data and maintained only at props

Screenshot_20230412_212045

### CODE REVIEW

File: src/Components/PreviewSignature/PreviewSignature.vue [hunk 1]
```diff
@@ -26,7 +26,6 @@ export default {
 		canvas: null,
 		isDrawing: false,
 		color: '#000000',
-		src: null,
```
Comment (enieber) [inline src/Components/PreviewSignature/PreviewSignature.vue:29]: this define src and no allow use in template?

Comment (vitormattos) [reply src/Components/PreviewSignature/PreviewSignature.vue:29]: src already was defined at `props`, the reported error is that isn't ok to use the same property name as `data`. I only removed from data and worked to me.

**Review by enieber — APPROVED** [review]

**Pull request merged**